<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImplantView</class>
 <widget class="QWidget" name="ImplantView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>291</width>
    <height>211</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ImplantView</string>
  </property>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Система:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ширина:</string>
   </property>
  </widget>
  <widget class="LineEdit" name="system_edit">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>10</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>141</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Време на имплантиране:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Вид имплантиране:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="length">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>100</y>
     <width>61</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> мм</string>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>100</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Дължина:</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="width">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>100</y>
     <width>71</width>
     <height>22</height>
    </rect>
   </property>
   <property name="suffix">
    <string> мм</string>
   </property>
   <property name="maximum">
    <double>20.000000000000000</double>
   </property>
   <property name="singleStep">
    <double>0.050000000000000</double>
   </property>
  </widget>
  <widget class="QComboBox" name="timeCombo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>40</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Отложено</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Ранно</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Имедиантно</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="typeCombo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>70</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Закрито</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Цикатризационен винт</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Временна корона</string>
    </property>
   </item>
  </widget>
  <widget class="QComboBox" name="tissueCombo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>130</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Roll flap</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Free gingival graft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Connective tissue graft</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>130</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Мекотъканна аугментация:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="boneCombo">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>160</y>
     <width>131</width>
     <height>22</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Xenograft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Allograft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Autograft</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Бета-3-калциев фосфат</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>160</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Костозаместител:</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="sinusCheck">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>190</y>
     <width>91</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Синус лифт</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="membrCheck">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>190</y>
     <width>81</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Мембрана</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>&quot;View/uiComponents/qt_derived/LineEdit.h&quot;</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
